package.json

```diff
@@ -107,6 +107,7 @@
     "fs-extra": "3.0.1",
     "history": "4.6.1",
     "html2canvas": "0.5.0-beta4",
+    "immutable": "4.0.0-rc.12",
```

Comment:
needed by the new react-grid?

Reply:
I think is because of this issue, closed but still very popular:
https://github.com/adazzle/react-data-grid/issues/80

Reply:
Ok, got it